package.json

```
@@ -52,6 +53,7 @@
     "@carbon/import-once": "10.4.0",
     "@carbon/layout": "10.16.0",
     "@carbon/motion": "10.11.0",
+    "@carbon/telemetry": "^0.0.0-alpha.6",
```

```suggestion
    "@carbon/telemetry": "0.0.0-alpha.6",
```

I'd pin this before we hit 1.0 just in case 👀